src/network/mcpe/handler/InGamePacketHandler.php

```diff
@@ -681,6 +686,7 @@ private function handlePlayerActionFromData(int $action, BlockPosition $blockPos
 
 		switch($action){
```

Comment:
As far as I researched CRACK_BREAK & ACTION_BREAK_BLOCK are no longer used and can thus be removed